treeherder/services/pulse/consumers.py

```diff
@@ -162,6 +162,36 @@ def on_message(self, body, message):
         )
         message.ack()
 
+class PulsesConsumer(PulseConsumer):
```

Comment:
I would have preferred to call it `PulseConsumer` but that is already taken. Can you please name it `JointConsumer`?

Reply:
Okay!